src/GovUk.Education.ExploreEducationStatistics.Content.Search.FunctionApp/Functions/CommandHandlers/FullReset/FullResetFunction.cs

```diff
@@ -0,0 +1,35 @@
+﻿using GovUk.Education.ExploreEducationStatistics.Content.Search.FunctionApp.Functions.CommandHandlers.RefreshSearchableDocument.Dto;
+using GovUk.Education.ExploreEducationStatistics.Content.Search.FunctionApp.Services;
+using GovUk.Education.ExploreEducationStatistics.Content.Search.FunctionApp.Services.ResetSearchableDocuments;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Azure.Functions.Worker;
+
+namespace GovUk.Education.ExploreEducationStatistics.Content.Search.FunctionApp.Functions.CommandHandlers.FullReset;
+
+public class FullResetFunction(IFullSearchableDocumentResetter fullSearchableDocumentResetter)
+{
+    [Function(nameof(FullSearchableDocumentsReset))]
+    public async Task<FullSearchableDocumentsResetOutput> FullSearchableDocumentsReset(
```

Comment:
I can't see any test class for this function. I know it would be pretty trivial but I think we've got unit tests for all the other functions.